src/MergeContext.js

```diff
@@ -14,6 +14,8 @@ class MergeContext {
         this._shaLimit = 6;
         // the remainder (>0) of the min or max voting delay (in ms)
         this._votingDelay = null;
+        // information used for fake approval status creation/updating
+        this._approvalResult = {params: null, delayMs: -1};
```

Comment:
Please create an Approval class (with a proper constructor, etc.) instead of using these nested anonymous structs.